### PR TITLE
OPSEXP-1635 add elasticsearch role

### PR DESCRIPTION
**Pull request by gionn**
OPSEXP-1635

### CODE REVIEW

File: roles/elasticsearch/tests/inventory [hunk 1]
```diff
@@ -0,0 +1,2 @@
+localhost
```
Comment (alxgomz) [inline roles/elasticsearch/tests/inventory:1]: What's this `tests` folder for?

File: roles/elasticsearch/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
```
Comment (alxgomz) [inline roles/elasticsearch/tasks/main.yml:1]: these tasks require `become: true`

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: roles/elasticsearch/meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  license: Apache-2.0
+
+  min_ansible_version: "2.12"
+
+  platforms:
+    - name: Ubuntu
+      versions:
+        - focal
+        - bionic
+    - name: EL
```
[the remark this answers is not in the document]
Comment (gionn) [reply roles/elasticsearch/meta/main.yml:16]: the available platforms are the ones returned by the galaxy api https://galaxy.ansible.com/api/v1/platforms/ (there is a search that returns 0 results for rocky, and for centos still return EL)

not sure it really matters what we put here if we don't publish this role on galaxy, I tried to fill everything as best as I can

File: roles/elasticsearch/tasks/main.yml (hunk 2 above)
Comment (gionn) [reply roles/elasticsearch/tasks/main.yml:1]: here I have broken the deprivileged molecule tests and noticed only today when working on the new pr, will fix

File: roles/elasticsearch/tests/inventory (hunk 1 above)
Comment (gionn) [reply roles/elasticsearch/tests/inventory:1]: good catch, I didn't notice before, they have been created automatically by running `molecule init role`, will try to remove them

File: roles/elasticsearch/meta/main.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (alxgomz) [reply roles/elasticsearch/meta/main.yml:16]: good point... That doesn't really match our supported and could give the false impression it works on platforms where it actually doesn't but better stick with what galaxy knows and understand

**Review by alxgomz — APPROVED** [review]
LGTM

**Pull request merged**